db/pluralCategories.json

```diff
@@ -500,6 +500,7 @@
     ],
     "mt": [
         "one",
+        "two",
```

Comment:
yes, Maltese has a 'two` plural category. I remember that I created to ticket for that to cldr before :)